packages/client/hmi-client/src/workflow/ops/model-config/tera-model-config.vue

```diff
@@ -263,6 +263,30 @@
 			/>
 		</tera-drilldown-section>
 	</tera-drilldown>
+
+	<Teleport to="body">
+		<tera-modal v-if="sanityCheckErrors.length > 0">
+			<template #header>
+				<h4>Warning, these settings may cause errors</h4>
+			</template>
+			<template #default>
+				<section style="max-height: 22rem; overflow-y: scroll">
+					<div v-for="(errString, idx) of sanityCheckErrors" :key="idx">
+						{{ errString }}
+					</div>
+				</section>
+			</template>
+			<template #footer>
+				<Button label="Ok" class="p-button-primary" @click="sanityCheckErrors = []" />
+				<Button
+					label="Ignore"
```

Comment:
I feel the label `Ignore` to be a bit confusing by itself. Maybe it's just me.
```suggestion
					label="Ignore warnings and use configuration"
```